### PR TITLE
static screens and login signup components added

### PR DESCRIPTION
**Pull request by abhishektanwar**
Static screens added for :
-VIdeo Explore page
- Playlist page
- History page
- Liked videos page
- Login ,signup modal
- Modal wrapper component

Files to check : 
- src/App.js
- src/Contexts/ModalContext/index.jsx
- src/components/ModalWrapper/index.jsx
- src/components/AsideNav/index.jsx
- src/components/Authentication/Login.jsx
- src/components/Authentication/SignUp.jsx
- src/components/VideoListingGrid/index.jsx
- src/components/VideoListingGrid/video-listing-grid.css
- src/screens/History/index.jsx
- src/screens/LikedVideos/index.jsx
- src/screens/Playlists/index.jsx

### CODE REVIEW

File: src/components/AsideNav/aside-nav.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
   width:10vw;
   border:1px solid green;
   background-color: var(--grey-background-color);
+  height: 100vw;
```
Comment (sanjitsarkar) [inline src/components/AsideNav/aside-nav.css]: **Suggestion:** we should use vh for height.

File: src/components/Authentication/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  const handleLoginCredentialInput = (e) => {
+    console.log([e.target.name], e.target.value);
+    setLoginCredentials((creds) => ({
+      ...creds,
+      [e.target.name]: e.target.value,
+    }));
+  };
+
+  const loginUser = async () => {
+    // e.preventDefault();
```
Comment (sanjitsarkar) [inline src/components/Authentication/Login.jsx]: **Suggestion:** remove commented and console.log in production.

File: src/components/Authentication/authentication.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+.card-action-btn-container a{
+	margin-top: 20px;
+	justify-content: center;
+	display: flex;
+	text-decoration: none;
+	color: inherit;
+}
+
+.auth-body{
+	height: calc(100vh - 10vh);
```
Comment (sanjitsarkar) [inline src/components/Authentication/authentication.css]: **Suggestion:** you can simply put 90vh without using calc function.

File: src/components/CardSlider/index.jsx [hunk 4]
```diff
@@ -34,7 +34,7 @@ const CardSlider = (props) => {
         {[1,2,3,4,5,6,7,8,9,10,11].map((item)=>{
```
Comment (sanjitsarkar) [inline src/components/CardSlider/index.jsx:34]: **Suggestion:** Try using [...Array(11)] in place of [1,2,3,4,5,6,7,8,9,10,11] as for larger values. So, we don't have to type out all the numbers.

File: src/components/ModalWrapper/modal-wrapper.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  justify-content: center;
+  align-items: center;
+  z-index: 20;
+}
+
+.modal-wrapper{
+  width: 800px;
+  height: 500px;
+  box-shadow: 0 5px 16px rgba(0,0,0,0.2);
+  background: #fff;
```
Comment (sanjitsarkar) [inline src/components/ModalWrapper/modal-wrapper.css:16]: **Suggestion:** use variable for color and use rem for larger value than 10px.

File: src/components/VideoListingGrid/index.jsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import VideoCard from '../VideoCard'
+import './video-listing-grid.css'
+const VideoListingGrid = () => {
+  return (
+    <section className="video-listing">
+        {[1,2,3,4,5,6,7,8,9].map((item)=> {return (<VideoCard variant="vertical" />)})}
```
Comment (sanjitsarkar) [inline src/components/VideoListingGrid/index.jsx:7]: **Suggestion:** Here also you can use [...Array(9)]

**Review by sanjitsarkar — APPROVED** [review]

File: src/components/Authentication/authentication.css [hunk 7]
```diff
@@ -0,0 +1,61 @@
+.authentication-container{
+	background-color: var(--bg-color-primary);
+	height:fit-content;
+	width: 50em;
```
Comment (itsmebipul00) [inline src/components/Authentication/authentication.css]: Suggestion: You could use rems here for consistency. Please let me know if you are using it for a case

**Review by itsmebipul00 — APPROVED** [review]
Good to merge, if possible with the suggested changes

File: src/components/AsideNav/aside-nav.css (hunk 1 above)
Comment (abhishektanwar) [reply src/components/AsideNav/aside-nav.css]: yes, this was not required. thanks

File: src/components/Authentication/Login.jsx (hunk 2 above)
Comment (abhishektanwar) [reply src/components/Authentication/Login.jsx]: removed

File: src/components/Authentication/authentication.css (hunk 3 above)
Comment (abhishektanwar) [reply src/components/Authentication/authentication.css]: fixed

File: src/components/CardSlider/index.jsx (hunk 4 above)
Comment (abhishektanwar) [reply src/components/CardSlider/index.jsx:34]: Thank you for mentioning this.

File: src/components/Authentication/authentication.css (hunk 7 above)
Comment (abhishektanwar) [reply src/components/Authentication/authentication.css]: This was a mistake. Thank you for pointing it out.

**Pull request merged**